mon-pix/translations/fr.json

```diff
@@ -1174,6 +1174,9 @@
         "activity": {
           "verify": "Vérifier"
         },
+        "element": {
+          "alternativeInstruction": "Afficher l'alternative textuelle"
```

Comment:
c'est un peu dérangeant qu'en français on parle d'`alternative textuelle` pour le bouton, alors que notre ressource possède la propriété `alternativeInstruction`

Reply:
"_Afficher l'instruction alternative_" ?

Reply:
Je voulais dire pour la clé et le nom du champ `alternativeInstruction`. Le libellé est correct

Reply:
un autre petit chose - pour le libellé entre anglais et francais - on utilise `l'alternative textuelle` et `alternative instruction` peut-etre le nommage du champ est aussi liée de ça

Reply:
+1 @mirawlin

Reply:
Discuté avec @Jeremiejade et @nlepage : on utilisera pas le modèle `Challenge` dans tous les cas donc on peut faire au plus clean pour notre usage.

Pour l'anglais : on utilise "Hide alternative text"/"Show alternative text"
 ![image](https://github.com/1024pix/pix/assets/5855339/6b3cfbee-1c4f-4397-9de7-430e778a52e6)

Reply:
on est ok qu'on fait pas de toggle nous, donc juste "_Show / display_ (au choix) _alternative text_" ?

Reply:
Yes !

Reply:
oui je suis en train de push le changement - mais j'ai laissé le nom de champ as is pour le moment

Reply:
Je pense qu'on peut renommer le champ aussi !